ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -59,162 +49,159 @@ func NewSetupMixerAccountsPage(l *load.Load) *SetupMixerAccountsPage {
 // the page is displayed.
 // Part of the load.Page interface.
 func (pg *SetupMixerAccountsPage) OnNavigatedTo() {
-	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
+	wl := load.NewWalletMapping(pg.WL.SelectedWallet.Wallet)
+	accts, err := wl.GetAccountsRaw()
+	if err != nil {
+		log.Errorf("Unable to get accounts to set up mixer: %v", err)
+		return
+	}
+	pg.manualEnabled = len(accts.Accounts) > 2
 }
 
 // Layout draws the page UI components into the provided layout context
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *SetupMixerAccountsPage) Layout(gtx layout.Context) layout.Dimensions {
-	body := func(gtx C) D {
-		page := components.SubPage{
-			Load:       pg.Load,
-			Title:      values.String(values.StrSetUpNeededAccs),
-			BackButton: pg.backButton,
-			Back: func() {
-				pg.ParentNavigator().CloseCurrentPage()
-			},
-			Body: func(gtx C) D {
-				return pg.Theme.Card().Layout(gtx, func(gtx C) D {
-					gtx.Constraints.Min.X = gtx.Constraints.Max.X
-					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
-						layout.Flexed(1, func(gtx C) D {
-							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+	return layout.Inset{Top: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+		return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Center.Layout(gtx, func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding25}.Layout(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
```

Comment:
Using the linear layout widget will technically remove these lines of code as you can align the layout by passing the necessary params.

Reply:
Was able to remove the Flex further above like you suggested in another place. 

Linear in place of Center?